libExt/CompilerInstanceCtu.h

```diff
@@ -177,7 +177,11 @@ class CompilerInstanceCtu : public CompilerInstance {
     if(cf)
     if (cf->BeginSourceFile(*this, FIF)) {
       setFrontendFileStatus();
-      cf->Execute();   
+      auto exec_result = cf->Execute();
```

Comment:
This one was throwing an "ignored return value" warning. Anything useful that can be done if the execute fails?